.github/workflows/ci.yml

```diff
@@ -0,0 +1,320 @@
+        matrix.os == 'macos-latest' &&
+        matrix.vtk_version == 'v9.0.1'
+      working-directory: ${{github.workspace}}/dependencies/vtk
+      run: |
+        sed -i '' '57i\'$'\n''herr_t H5O__fsinfo_set_version(H5F_t *f, H5O_fsinfo_t *fsinfo);' ThirdParty/hdf5/vtkhdf5/src/H5Fsuper.c
+        sed -i '' '46i\'$'\n''#include "H5CXprivate.h"' ThirdParty/hdf5/vtkhdf5/src/H5Oint.c
+        sed -i '' '36i\'$'\n''#include "H5CXprivate.h"' ThirdParty/hdf5/vtkhdf5/src/H5Rint.c
+
+    - name: Configure VTK
+      if: steps.cache-vtk.outputs.cache-hit != 'true'
+      working-directory: ${{github.workspace}}/dependencies/vtk_build
+      run: >
+        cmake ../vtk
+        -DVTK_ENABLE_LOGGING=OFF
+        -DVTK_BUILD_TESTING=OFF
+        -DCMAKE_BUILD_TYPE=Release
+        -DVTK_LEGACY_REMOVE=ON
+        -DBUILD_SHARED_LIBS=OFF
+        -DCMAKE_INSTALL_PREFIX:PATH=../vtk_install
+        -DVTK_GROUP_ENABLE_Rendering=DEFAULT
+        -DVTK_GROUP_ENABLE_StandAlone=DEFAULT
+        -DVTK_MODULE_ENABLE_VTK_CommonSystem=YES
+        -DVTK_MODULE_ENABLE_VTK_FiltersGeneral=YES
+        -DVTK_MODULE_ENABLE_VTK_FiltersGeometry=YES
+        -DVTK_MODULE_ENABLE_VTK_ImagingCore=YES
+        -DVTK_MODULE_ENABLE_VTK_ImagingHybrid=YES
+        -DVTK_MODULE_ENABLE_VTK_InteractionStyle=YES
+        -DVTK_MODULE_ENABLE_VTK_InteractionWidgets=YES
+        -DVTK_MODULE_ENABLE_VTK_IOCityGML=YES
+        -DVTK_MODULE_ENABLE_VTK_IOExodus=YES
+        -DVTK_MODULE_ENABLE_VTK_IOGeometry=YES
+        -DVTK_MODULE_ENABLE_VTK_IOImage=YES
+        -DVTK_MODULE_ENABLE_VTK_IOImport=YES
+        -DVTK_MODULE_ENABLE_VTK_IOParallel=YES
+        -DVTK_MODULE_ENABLE_VTK_IOPLY=YES
+        -DVTK_MODULE_ENABLE_VTK_IOXML=YES
+        -DVTK_MODULE_ENABLE_VTK_RenderingAnnotation=YES
+        -DVTK_MODULE_ENABLE_VTK_RenderingCore=YES
+        -DVTK_MODULE_ENABLE_VTK_RenderingLabel=YES
+        -DVTK_MODULE_ENABLE_VTK_RenderingOpenGL2=YES
+        -DVTK_MODULE_ENABLE_VTK_RenderingVolumeOpenGL2=YES
+        -DVTK_MODULE_ENABLE_VTK_jsoncpp=YES
+        ${{ matrix.os == 'windows-latest' && '-Ax64 -DCMAKE_POLICY_DEFAULT_CMP0091=NEW -DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded' || null }}
+
+    - name: Build VTK
+      if: steps.cache-vtk.outputs.cache-hit != 'true'
+      working-directory: ${{github.workspace}}/dependencies/vtk_build
+      run: cmake --build . --parallel 2 --target install --config Release
+
+    - name: Cache LFS data
+      id: cache-lfs
+      uses: actions/cache@v2
+      with:
+        path: lfs_data
+        key: lfs-data-fa770f401af4d5760f17eedeb0325a5875f3a328
+
+    - name: Checkout LFS data
+      if: steps.cache-lfs.outputs.cache-hit != 'true'
+      uses: actions/checkout@v2
+      with:
+        path: 'lfs_data'
+        fetch-depth: 0
+        lfs: 'true'
+
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        path: 'source'
+        fetch-depth: 0
+        lfs: 'false'
+
+    - name: Setup LFS data ubuntu
```

Comment:
Can you use a CMake command to factorize the copy step?  
Something like `cmake -E copy_directory ./lfs_data/data ./source/data`

Reply:
good idea, I will try it